packages/app/src/app/pages/Dashboard/Components/Selection/ContextMenus/RepositoryMenu.tsx

```diff
@@ -54,19 +60,28 @@ export const RepositoryMenu: React.FC<RepositoryMenuProps> = ({
       position={position}
       style={{ width: 120 }}
     >
-      <MenuItem onSelect={() => history.push(repositoryUrl)}>
-        Open repository
+      <MenuItem
+        onSelect={() => {
+          window.location.href = defaultBranchUrl;
+        }}
+      >
+        Open {quotes(providerRepository.defaultBranch)} branch
       </MenuItem>
-      <MenuItem onSelect={() => window.open(repositoryUrl, '_blank')}>
-        Open repository in a new tab
+      <MenuItem onSelect={() => window.open(defaultBranchUrl, '_blank')}>
+        Open {quotes(providerRepository.defaultBranch)} in a new tab
       </MenuItem>
       <MenuItem onSelect={() => window.open(githubUrl, '_blank')}>
         Open on GitHub
       </MenuItem>
+
       <Menu.Divider />
+
       <MenuItem onSelect={() => window.open(branchFromDefaultUrl, '_blank')}>
         Create branch
       </MenuItem>
+      <MenuItem onSelect={() => history.push(repositoryUrl)}>
+        See all branches
+      </MenuItem>
```

Comment:
Would love to see these be changed into [`MenuLink`s](https://reach.tech/menu-button/#menulink) :)

Reply:
https://www.notion.so/codesandbox-app/Tech-debt-accounting-c4e9e7420721494faa22b47cbb35cc10?p=a9e05cf80fb147a8b71d72a3a165a0ca&pm=s